src/click/core.py

```diff
@@ -1935,6 +1935,9 @@ def _write_opts(opts):
             else:
                 default_string = self.default
             extra.append(f"default: {default_string}")
+            if isinstance(self.type, IntRange):
```

Comment:
Still needs to be un-indented, it became part of the previous block when you removed the `if`.

Reply:
Took a bit more time to make the change this time, hope there aren't any small bugs. In general for a simple feature is on test where it tries to send good data (like not a test case trying to crash it) good enough? Thanks